Formula/n/ns-3.rb

```diff
@@ -26,6 +27,12 @@ class Ns3 < Formula
   uses_from_macos "libxml2"
   uses_from_macos "sqlite"
 
+  # cmake patch, upstream PR patch, https://gitlab.com/nsnam/ns-3-dev/-/merge_requests/1993
+  patch do
+    url "https://gitlab.com/nsnam/ns-3-dev/-/commit/08a993d3b48fd219fc3d4a3bf203dc23a910e76a.diff"
+    sha256 "6043985791177f1fc953224db47c96d0b35fd4d38b52a7365ce06826825cebc8"
+  end
+
```

Comment:
this has been released with ns-3 3.42, https://github.com/Homebrew/homebrew-core/pull/173511